package com.atguigu.chapter13

import scala.util.control.Breaks._

object Test01 {
  def main(args: Array[String]): Unit = {

//    var i : Double = 10 / 3
//    println(i.formatted("%.2f"))
//
//    var num = 10
//    num -= 10
//    num += 20
//    println(num)

//    for(x <- 1 to 10 if x % 2 == 1 ){
//      println(x)
//    }

//    for(x <- 1 to 10 by 3){
//      println(x)
//    }

//    val res = for(x <- 1 to 10) yield x * 2 +3
//    println(res)

//
//    for(x <- 9 to 1 by -2){
//      print(x)
//    }
//
//    println()
//
//    for(x <- 1 to 10 by 2 reverse){
//      println(x)
//    }
//    try{
//      for(x <- List(1,10)){
//        if(x == 1) throw new ArithmeticException
//      }
//    }catch{
//      case e:Exception => println("error")
//    }
//    println("hello")

//    breakable(
//      for(x <- List(2,1,10)) {
//        if(x == 1) break
//        println(x)
//      }
//    )
//
//    println("hello")



  }
}
